.bazelversion

```diff
@@ -1 +1 @@
-7.6.1
+7.6.2
```

Comment:
Why update to 7.6.2 and not the latest 7.x (i.e. 7.7.1) or even 8.x?

Reply:
7.7.1 was published only few days ago. My long term goal is to turn envoy in bzlmod and it is yet in 7.6.2 so I just need to ensure actual compatibility